docs/source/analyze/mysql.md

```diff
@@ -0,0 +1,66 @@
+---
+title: MySQL
+description: Check version and connection status 
+---
+
+The `MySQL` analyzer is available to check version and connection status of a MySQL database. It relies on the data collected by the [MySQL collector](/collect/mysql/).
```

Comment:
Can you put one sentence per line? It makes reviews and changes easier in the future.

Reply:
Done!